src/components/KeyboardToolbar/index.tsx

```diff
@@ -73,6 +73,9 @@ const TEST_ID_KEYBOARD_TOOLBAR_DONE = `${TEST_ID_KEYBOARD_TOOLBAR}.done`;
 
 const KEYBOARD_TOOLBAR_HEIGHT = 42;
 const DEFAULT_OPACITY: HEX = "FF";
+const KEYBOARD_HAS_ROUNDED_CORNERS =
+  Platform.OS === "ios" && parseInt(Platform.Version, 10) >= 26;
+const OPENED_OFFSET = KEYBOARD_HAS_ROUNDED_CORNERS ? -10 : 0;
```

Comment:
could you tell me how did you measure that it should be -10?

Reply:
Hmm, borders are different. Looks like apple uses squircle for this element. Have you tried to use borderCurve style prop?